.github/workflows/main.yml

```diff
@@ -40,44 +45,19 @@ jobs:
     needs: check-workflows
     uses: ./.github/workflows/lint-build-test.yml
 
-  is-release:
```

Comment:
I wonder if we should update `MetaMask/action-is-release` to have a lint option instead, maybe 🤔 But for now it's fine to have it just in `core`, so we can finetune it if necessary.

Reply:
Also wondering if `validate-release` should completely replace `MetaMask/action-is-release`?